server/serializers/subscriptions.go

```diff
@@ -163,6 +163,20 @@ type Resource struct {
 	PullRequest   PullRequest  `json:"pullRequest"`
 	Commits       []Commit     `json:"commits"`
 	RefUpdates    []RefUpdates `json:"refUpdates"`
+	Fields        Fields       `json:"fields"`
+	Revision      Revision     `json:"revision"`
+}
+
+type Revision struct {
+	Fields Fields `json:"fields"`
+}
+
+type Fields struct {
```

Comment:
If everything inside this struct is related to System its better to name this SystemFields I think

Reply:
I am using the same name returned from the API